CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.6.5 (2021-04-26)
+
+- Fixed the tab close buttons to follow proper accessibility guidelines. [#162](https://github.com/blackbaud/skyux-tabs/pull/162)
```

Comment:
```suggestion
- Fixed the tab close button to follow accessibility guidelines. [#162](https://github.com/blackbaud/skyux-tabs/pull/162)
```
I think singular is correct here for the button, but let me know if I'm mistaken. And "proper" just seems like an unnecessary qualifier here.